dtfs-central-api/src/v1/routes/portal-routes.js

```diff
@@ -149,6 +149,8 @@ portalRouter.route('/deals/:id').get(getDealController.findOneDealGet);
  *             example:
  *               user: { _id: '123456abc' }
  *               dealUpdate: { aNewField: true }
+ *               auditDetails:
```

Comment:
sadly this ref should be on the schema itself, not in the example - could you rewrite these docs?